util/jaeger_helper.go

```diff
@@ -0,0 +1,60 @@
+package util
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/kiali/kiali/config"
+)
+
+type Trace struct {
```

Comment:
these two structs are also defined in services.go, I guess only one definition is needed?

Reply:
ups i forgot removed from here...I am going so fast...sorry:'(